### PR TITLE
Service ticket detail page - first page

### PR DESCRIPTION
**Pull request by Georgi2704**

### CODE REVIEW

File: apps/wfo-ui-surf/components/WfoServiceTicketDetailPage/utils.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { EuiIcon } from '@elastic/eui';
+
+// const MAX_LABEL_LENGTH = 45;
```
Comment (DutchBen) [inline apps/wfo-ui-surf/components/WfoServiceTicketDetailPage/utils.tsx:4]: Uncomment or remove

**Review by DutchBen — APPROVED** [review]

File: apps/wfo-ui-surf/components/WfoServiceTicketsList/WfoServiceTicketsList.tsx [hunk 2]
```diff
@@ -124,24 +121,24 @@ export const WfoServiceTicketsList = ({
             field: SERVICE_TICKET_FIELD_START_DATE,
             name: t('startDate'),
             width: '150',
-            render: (date) => <WfoDateTime dateOrIsoString={date} />,
-            renderDetails: parseIsoString(parseDateToLocaleDateTimeString),
-            clipboardText: parseIsoString(parseDateToLocaleDateTimeString),
+            render: (date) => (
+                <span>{parseDateToLocaleDateTimeString(new Date(date))}</span>
```
[the remark this answers is not in the document]
Comment (ricardovdheijden) [reply apps/wfo-ui-surf/components/WfoServiceTicketsList/WfoServiceTicketsList.tsx:125]: +1

File: apps/wfo-ui-surf/components/WfoServiceTicketDetailPage/WfoServiceTicketDropdownItems.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        const currentPath = router.asPath;
+        router.push(`${currentPath}/new-email`);
+    };
+
+    return (
+        <EuiFlexGroup
+            direction={'column'}
+            alignItems={'flexStart'}
+            gutterSize={'none'}
+            style={{ paddingRight: theme.size.xxxxl }}
```
Comment (ricardovdheijden) [inline apps/wfo-ui-surf/components/WfoServiceTicketDetailPage/WfoServiceTicketDropdownItems.tsx:33]: Try to stick with the css prop to pass custom style.

File: apps/wfo-ui-surf/components/WfoServiceTicketDetailPage/utils.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { EuiIcon } from '@elastic/eui';
+
+export enum ServiceTicketTabIds {
+    GENERAL_TAB = 'general-id',
+    NOTIFICATION_LOG = 'notification-log-id',
+    SENT_EMAILS = 'sent-emails-id',
+}
+
+export const tabs = [
```
Comment (ricardovdheijden) [inline apps/wfo-ui-surf/components/WfoServiceTicketDetailPage/utils.tsx:10]: I would like to suggest to not use something like a utils.tsx. Preferably only .ts files. That means you will have to move this object to the component where it is actually used.

File: apps/wfo-ui-surf/components/WfoServiceTicketsList/WfoServiceTicketsList.tsx [hunk 5]
```diff
@@ -124,24 +122,24 @@ export const WfoServiceTicketsList = ({
             field: SERVICE_TICKET_FIELD_START_DATE,
             name: t('startDate'),
             width: '150',
-            render: (date) => <WfoDateTime dateOrIsoString={date} />,
-            renderDetails: parseIsoString(parseDateToLocaleDateTimeString),
-            clipboardText: parseIsoString(parseDateToLocaleDateTimeString),
+            render: (date) => (
+                <span>{parseDateToLocaleDateTimeString(parseDate(date))}</span>
```
Comment (ricardovdheijden) [inline apps/wfo-ui-surf/components/WfoServiceTicketsList/WfoServiceTicketsList.tsx:126]: if the span is not required you can omit that. Also there is another utility function that can do what you are doing here:

packages/orchestrator-ui-components/src/utils/date.ts --> formatDate

Line 125-127 (the render function) can be replaced with:
`render: formatDate,`

Same applies to all renders of dates here.

**Review by ricardovdheijden — APPROVED** [review]

**Pull request merged**